hc/accounts/models.py

```diff
@@ -86,3 +87,8 @@ def invite(self, user):
 class Member(models.Model):
     team = models.ForeignKey(Profile)
     user = models.ForeignKey(User)
+
+
+class MemberAllowedChecks(models.Model):
+    user = models.ForeignKey(User)
+    checks = models.ForeignKey(Check)
```

Comment:
I don't grok why this field name is pluralized. This also goes for the model name.